packages/x-charts/src/models/axis.ts

```diff
@@ -391,6 +391,10 @@ type CommonAxisConfig<S extends ScaleName = ScaleName, V = any> = {
    * - function: Receives the calculated extremums as parameters, and should return the axis domain.
    */
   domainLimit?: 'nice' | 'strict' | ((min: number, max: number) => { min: number; max: number });
+  /**
+   * If `true`, the axis will be ignored by the tooltip with `trigger='axis'`.
+   */
+  disableTooltipInteraction?: boolean;
```

Comment:
```suggestion
  hideTooltip?: boolean;
```

or 

```suggestion
  hideTooltip?: boolean;
```

or simply the same as the internal value, which would be clearer imo, though we would have to treat `undefined` as `true`

```suggestion
  triggerTooltip?: boolean;
```

Reply:
What about `ignoreTooltip`?

Reply:
The issue with `hideTooltip` is that it exist also in the series config, but the behavior is not exactly the same, so it might create some confusion

Reply:
What if we invert it and call it something like `showInTooltip` or `displayInTooltip` where the default is `true`? 

Alternatively, we could use `hideFromTootip` or `hideInTooltip`.